### PR TITLE
(WIP) Teams Documentation Page

### PR DESCRIPTION
**Pull request by DeadSuperHero**
(Not ready for review or merge just yet)
Currently a first-pass, needs content revisions. The goal is to demonstrate teams as a key part of organizational structure within Sentry, with examples of:

- What kinds of teams might an organization have?
- A project with multiple teams assigned to it vs a team with multiple projects assigned to it

### CODE REVIEW

**Comment by DeadSuperHero** [comment]
Paperdoc mockup: https://paper.dropbox.com/doc/OKR-Sentry-Docs-Projects--ATC34CwD8eRjapx6s8dFgFWxAg-irTxhDa8vQfG7vk5ulEMh

File: src/collections/_documentation/accounts/teams.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Teams
+sidebar_order: 2
+---
+
+Teams are a workflow feature in Sentry that reflect the various departments and initiatives within a company's engineering organization. They are used to denote two things: access, and
+structure. By leveraging teams, an organization can introduce a sense of hierarchy, and route engineering groups directly to the projects that they work on.
```
Comment (benvinegar) [inline src/collections/_documentation/accounts/teams.md]: The benefit of "introduce a sense of hierarchy" is unclear to me. What if we changed these sentences to:

> Teams are a workflow feature in Sentry that are designed to reflect the teams within a company's engineering organization. They are used to denote two things: access, and
responsibility. By leveraging teams, an organization can scope a team's access in Sentry to the services they work on, triage issues to teams and team members, and create ownership rules that ensure the right teams and team members are alerted to problems as they emerge.

Comment (benvinegar) [reply src/collections/_documentation/accounts/teams.md]: Aside: I'm really excited for this section!

**Comment by PeloWriter** [comment]
Please let me know if there's any further work to be completed here as the PR is quite dated.

**Pull request closed without merging**